tsdb/engine/tsm1/file_store.go

```diff
@@ -436,6 +436,16 @@ func (f *FileStore) Replace(oldFiles, newFiles []string) error {
 		}
 	}
 
+	fd, err := os.Open(f.dir)
+	if err != nil {
+		return err
+	}
+	defer fd.Close()
+
+	if err := fd.Sync(); err != nil {
+		return err
+	}
+
```

Comment:
This block looks like `syncDir` which is defined in `file_unix.go`, should that be used instead?